lib/lrama/grammar/rule_builder.rb

```diff
@@ -110,11 +110,16 @@ def process_rhs
           when Lrama::Lexer::Token::Ident
             @replaced_rhs << token
           when Lrama::Lexer::Token::Parameterizing
-            parameterizing = ParameterizingRules::Builder.new(token, @rule_counter, @lhs_tag, user_code, precedence_sym, line)
-            parameterizing.build.each do |r|
-              @parameterizing_rules << r
+            if parameterizing_resolver.defined?(token.s_value)
+              parameterizing_resolver.build_rules(token, @rule_counter, @lhs_tag, line)
+              @parameterizing_rules = @parameterizing_rules + parameterizing_resolver.rules
+              @replaced_rhs = @replaced_rhs + parameterizing_resolver.tokens
```

Comment:
[nits] Is this same with `@replaced_rhs << parameterizing_resolver.tokens` ? If so I prefer `<<` so that it matches with other lines.

Reply:
Indeed it is. Thanks for catching this! 
Fixed in the following commit:
- https://github.com/ruby/lrama/pull/285/commits/7add3f5780174f0881ab69bc83c26fcf1b7a6065